Lib/test/test_site.py

```diff
@@ -534,11 +537,21 @@ def test_startup_imports(self):
         # http://bugs.python.org/issue19209
         self.assertNotIn('copyreg', modules, stderr)
 
-        # http://bugs.python.org/issue19218
-        collection_mods = {'_collections', 'collections', 'functools',
-                           'heapq', 'itertools', 'keyword', 'operator',
-                           'reprlib', 'types', 'weakref'
-                          }.difference(sys.builtin_module_names)
+        if sys.platform == 'win32' and  locale.getpreferredencoding() == 'cp65001':
+            # https://bugs.python.org/issue36778
+            # If the default ansi codepage on Windows is 65001 (CP_UTF8)
+            # then cp65001.py is loaded.  When cp65001.py loads it loads
+            # additional modules that result in a less optimized startup.
+            # Ignore the unoptimized module loads for now.
+            collection_mods = {'_collections', 'itertools', 'types',
+                            'weakref'
+                            }.difference(sys.builtin_module_names)
+        else:
+            # http://bugs.python.org/issue19218>
```

Comment:
this is typo at the end of the URL

Reply:
Victor fixed this in https://github.com/python/cpython/pull/13230
The test change is not needed